rust/core-lib/src/plugins/manager.rs

```diff
@@ -403,6 +439,20 @@ impl PluginManagerRef {
 
     }
 
+    pub fn toggle_tracing(&self, enabled: bool) {
+        let dummy_view_id = ViewIdentifier::from("view-id-0");
+        let only_globals = true;
+        self.lock().request_plugins_sync(
+            dummy_view_id, only_globals, "tracing_config",
+            &json!({"enabled": enabled}));
+    }
+
+    pub fn collect_trace(&self) -> Vec<Result<Value, xi_rpc::Error>> {
+        let dummy_view_id = ViewIdentifier::from("view-id-0");
+        let only_globals = true;
```

Comment:
if we _do_ go this route, we want `only_globals` to be false. Syntect, for instance, is not currently a global plugin. (Each document has its own instance of syntect running, something we ultimately want to change)

Reply:
Gotcha.  I didn't understand what that variable meant so I guessed wrong :).